### DISCUSSION TITLE
Option to Choose between different osu!catch touch controls

### DISCUSSION THREAD

**Discussion by FalconTRM** [ppy/osu#27806] [Ideas]
I prefer the controls that RBQ wanted and features for it the people from the discussion I referenced about, than the new controls, we should be able to have an option to choose controls we want..

_Originally posted by @FalconTRM in https://github.com/ppy/osu/discussions/20237#discussioncomment-9004331_

**Comment by Joehuu** [comment]
Can probably be easily done after https://github.com/ppy/osu/pull/27790 (mania touch control skinning, to consider the code structure before other rulesets).

**Comment by FalconTRM** [comment]
> Can probably be easily done after #27790 (mania touch control skinning, to consider the code structure before other rulesets).

That would be awesome!

**Comment by FalconTRM** [comment]
Also maybe have an option to where the direction buttons should be at and the boost, whether its on the left or right side, depends on what we prefer.

**Comment by PYUDNG** [comment]
That will be nice if implemented for catch, but I'm not sure whether #27790 is still in-progress or just being stopped. What is going on?
